core/controllers/contributor_dashboard.py

```diff
@@ -362,7 +362,7 @@ def _get_reviewable_exploration_opportunity_summaries(
         # 3. Get the reviewable translation suggestion target IDs for the user.
         # 4. Get story exploration nodes in order, filtering for explorations
         # that have in review translation suggestions.
-        if topic_name is None:
+        if topic_name is None or topic_name == 'All':
```

Comment:
Can you make 'All' a named constant, and have this (and any other usages) point to that constant?

Reply:
FYI: There should already be a constant for this. See `TOPIC_SENTINEL_NAME_ALL` in constants.ts.

Reply:
sg Sean! Thanks for the pointer Sagang!